lib/dal/src/attribute/prototype/debug.rs

```diff
@@ -131,178 +131,157 @@ impl AttributePrototypeDebugView {
             AttributeValue::component_prototype_id(ctx, attribute_value_id)
                 .await?
                 .is_some();
-        let destination_component_id =
-            AttributeValue::component_id(ctx, attribute_value_id).await?;
+        let component_id = AttributeValue::component_id(ctx, attribute_value_id).await?;
```

Comment:
We no longer have `expected_source_component_id`, so we don't need to mark this as a `destination_`.